moesifwsgi/moesif_data_holder.py

```diff
@@ -0,0 +1,61 @@
+from datetime import datetime
+import time
+import uuid
+
+class DataHolder(object):
+    """Capture the data for a request-response."""
+    def __init__(self, capture_transaction_id, id, method, url, ip, user_id, company_id, metadata, session_token, request_headers, content_length, request_body, transfer_encoding):
+        self.request_id = id
+        self.method = method
+        self.url = url
+        self.ip_address = ip
+        self.user_id = user_id
+        self.company_id = company_id
+        self.metadata = metadata
+        self.session_token = session_token
+        self.request_headers = request_headers
+        self.content_length = content_length
+        self.request_body = request_body
+        self.transfer_encoding = transfer_encoding
+        self.status = -1
+        self.response_headers = None
+        self.response_chunks = None
+        self.response_body_data = None
+        self.request_time = datetime.utcnow().strftime("%Y-%m-%dT%H:%M:%S.%f")[:-3]
+        self.start_at = time.time()
+        self.transaction_id = None
+        if not capture_transaction_id:
```

Comment:
when capture_transaction_id is true, you want to capture transaction id?
If that is the case why do u have 'if not' here. Is the variable name 'capture_transaction_id' doesn't mean what it says?

Reply:
Have renamed the variable name